cinnabar/plotting.py

```diff
@@ -35,6 +35,7 @@ def _master_plot(
     bootstrap_x_uncertainty: bool = False,
     bootstrap_y_uncertainty: bool = False,
     statistic_type: str = "mle",
+    scatter_kwargs: dict = {},
```

Comment:
Would it be worth keeping s=10 and marker='o' as the defaults here? That way you have the same behaviour as before just with a little bit extra control?

Reply:
Done!